modules/openstack/user.go

```diff
@@ -76,8 +82,9 @@ func (o *OpenStack) CreateUser(
 func (o *OpenStack) GetUser(
 	log logr.Logger,
 	userName string,
+	domainID string,
```

Comment:
Maybe if we need to change the function signature here anyway, we could pass in the User struct here? Which might improve the maintainability of this in the future if we need to add other parameters to it?

Reply:
We can update the sigunature but that would eventually affect all of the functions we have in this openstack module, which can affect a few more operators. So I tend to leave it now but am open for opinion from the others.

Reply:
yes, passing in a user struct would help to easy extend it in future without the immediate change of the consumers. if my grep was correct, only keystone, glance and heat use this module direct. but I guess we could also do it later